src/app/components/SearchBar/SearchBar.tsx

```diff
@@ -91,6 +98,10 @@ const SearchBar: FunctionComponent<SearchBarProps> = ({
     placeholder: placeHolder,
   }
 
+  const toggleWidth = (isOpen: boolean): void => {
```

Comment:
No need to add `: void` here as the type can be inferred.`
```suggestion
  const toggleWidth = (isOpen: boolean) => {
```